src/components/Header.tsx

```
@@ -1,21 +1,55 @@
-        </button>
+
+      <div className="flex-none">
+        <div className="flex items-center gap-2">
+          <ThemeToggle />
+
+          {isAuthenticated ? (
+            <button
+              onClick={onLogout}
+              className="btn btn-ghost btn-sm gap-2"
+              title="Sair"
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)